MdePkg/Library/StackCheckLibNull/StackCheckLibNull.c

```diff
@@ -0,0 +1,16 @@
+/** @file
+  Defines the stack cookie variable for GCC, Clang and MSVC compilers.
+
+  Copyright (c) Microsoft Corporation. All rights reserved.
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+**/
+
+#include <Uefi.h>
+
+#if defined (__GNUC__) || defined (__clang__)
+VOID  *__stack_chk_guard = (VOID *)0x0;
+#elif defined (_MSC_VER)
+UINTN  __security_cookie = 0x0;
+#else
+  #error "Unsupported compiler"
+#endif

```

Comment:
Is the intent of the NULL instance for pipeline build only? Otherwise, should we always err the build when it is used?

Reply:
The pipeline will use the non-null version but I agree that generating a build error in the null instance is not the best route. I removed this error.